### PR TITLE
make day for conditional tag parsing configurable

### LINKED ISSUE #1815: Make conditional tag parser date configurable
For tests and other use cases we need to set the date that the parser uses as current date.

### PR DESCRIPTION
**Pull request by karussell**
fixes #1815

### CODE REVIEW

**Comment by easbar** [comment]
Cool thanks! The measurement failed with
```shell
 Exception in thread "main" java.lang.IllegalStateException: Cannot load properties to fetch EncodingManager configuration at: /ext_data/osm/bayern-190101.osm-gh/
219 	at com.graphhopper.routing.util.EncodingManager.create(EncodingManager.java:110)
220 	at com.graphhopper.GraphHopper.load(GraphHopper.java:742)
221 	at com.graphhopper.GraphHopper.importOrLoad(GraphHopper.java:617)
222 	at com.graphhopper.tools.Measurement.start(Measurement.java:146)
223 	at com.graphhopper.tools.Measurement.main(Measurement.java:67)
```
... no idea why

**Comment by karussell** [comment]
It seems that GraphHopper.init did not initialize the EncodingManager and so this code is entered which tries to create the EncodingManager from the properties file (and fails as the folder does not exist):

https://github.com/graphhopper/graphhopper/blob/master/core/src/main/java/com/graphhopper/GraphHopper.java#L740

File: benchmark/benchmark.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ mkdir -p ${SINGLE_RESULTS_DIR}
 # 1 - small map: node- and edge-based CH + slow routing
 java -cp tools/target/graphhopper-tools-*-jar-with-dependencies.jar com.graphhopper.tools.Measurement \
 datareader.file=${SMALL_OSM_MAP} \
-datareader.date_range_parser_day=2019-11-01
+datareader.date_range_parser_day=2019-11-01 \
```
Comment (easbar) [inline benchmark/benchmark.sh]: ah this happened to me multiple times :(

**Comment by easbar** [comment]
> Ah, that was my fault :D ... see the change

Ah I forgot this backslash also a few times :(

**Comment by easbar** [comment]
Now it worked (even though it shows as failing here, because I canceled one of the two duplicate builds)

**Pull request merged**